packages/liotad/README.md

```diff
@@ -1,23 +1,34 @@
-#Liota Package Manager
-Liota package manager consists of two parts: (1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and (2) A PackageMessengerThread which listens on one or more specific communication channels, which is a named pipe for now, to provide with an interface for users and automated agents to send commands to package manager.
+## Liota Package Manager
+
+Liota package manager consists of two parts:
+
+(1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and
```

Comment:
Instead of 'codes' it would be better to say either 'logic' or 'code'

Reply:
Changed 'codes' to 'logic'